react/components/ListDetails/ItemDetails.tsx

```diff
@@ -60,11 +60,13 @@ class ItemDetails extends Component<ItemDetailsProps, ItemDetailsState> {
 
     return (
       <div className="relative">
-        <div className="absolute top-0 left-0 ml5 mt6 pl2">
-          <Checkbox
-            checked={isSelected}
-            onChange={this.handleItemSelectedChange}
-          />
+        <div className="absolute ml4 mt4">
+          <label className="db ph3 pv3">
```

Comment:
Use `pa3` instead of `ph3 pv3`
```suggestion
          <label className="db pa3">
```